### PR TITLE
utilities/ScanBridge 

### PR DESCRIPTION
**Pull request by harshshah-tenable**
- Changes for ScanBridge Utility
- path: `tenable/utilities/scan_bridge.py`
- usage: `from tenable.utilities import ScanBridge`

image

### CODE REVIEW

File: tenable/utilities/scan_bridge.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            ... from tenable.io import TenableIO
+            ... from tenable.scimport TenableSC
+            ... tsc = TenableSC(username, password, url)
+            ... tio = TenableIO(access_key, secret_key, url)
+            ... sb = ScanBridge(tsc, tio)
+            ... sb.move(48,7)
+        '''
+        with open(f'{scan_id}.nessus', 'wb') as nessus:
+            self.tio.scans.export(scan_id, fobj=nessus)
+            # self.tio.v3.vm.scans.export(scan_id, fobj=nessus)
```
Comment (harshshah-tenable) [inline tenable/utilities/scan_bridge.py:58]: this would be updated once #52 is merged in develop

**Review by dhvanilshah-tenable — APPROVED** [review]

**Review by parvdesai-tenable — APPROVED** [review]

File: tenable/utilities/scan_bridge.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                will be imported.
+
+        Example:
+            >>> sb = ScanBridge(tsc, tio)
+            ... sb.bridge(48,7)
+        '''
+        try:
+            with open(f'{scan_id}.nessus', 'wb') as nessus:
+                self.tio.scans.export(scan_id, fobj=nessus)
+                # self.tio.v3.vm.scans.export(scan_id, fobj=nessus)
```
Comment (Mangusingh-te) [inline tenable/utilities/scan_bridge.py:65]: Remove dead-code.

File: tests/utilities/test_scan_bridge.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+# from responses import matchers
+
+
+dir_name = os.path.dirname(os.path.abspath(__file__))
+
+TXT_ATTACHMENT_FILE = 'test_file.txt'
+TXT_ATTACHMENT_FILE_PATH = os.path.join(dir_name, TXT_ATTACHMENT_FILE)
+
+SC_BASE_URL = r'https://cloud.tenablesc.com:443'
+# SCAN_BASE_URL = r'https://cloud.tenable.com/api/v3/scans'
```
Comment (Mangusingh-te) [inline tests/utilities/test_scan_bridge.py:24]: Remove dead-code.

**Review by Mangusingh-te — CHANGES_REQUESTED** [review]

File: tenable/utilities/scan_bridge.py (hunk 1 above)
Comment (alpesh-te) [reply tenable/utilities/scan_bridge.py]: I will take care with https://github.com/dhvanilshah-tenable/pyTenable/pull/52

File: tenable/utilities/scan_bridge.py (hunk 2 above)
Comment (alpesh-te) [reply tenable/utilities/scan_bridge.py:65]: I will take care with https://github.com/dhvanilshah-tenable/pyTenable/pull/52

File: tests/utilities/test_scan_bridge.py (hunk 3 above)
Comment (alpesh-te) [reply tests/utilities/test_scan_bridge.py:24]: I will take care with https://github.com/dhvanilshah-tenable/pyTenable/pull/52

**Review by alpesh-te — APPROVED** [review]
LGTM

**Pull request merged**